doc/tutorials/content/sources/spherical_voxel_grid/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
+
+project(voxel_grid)
+
+find_package(PCL 1.8.1 REQUIRED)
```

Comment:
This should be `1.8.1.99`. Reason: 1.8.1 is already released and it does not include your filter. We indicate "the next release after x.x.x" by appending `.99` suffix. When time comes to release the next version (whichever number it will have), we will search/replace all occurrences.